### PR TITLE
[Android] Take RenderTransform into account when calculating screen position

### PR DESCRIPTION
**Pull request by davidjohnoliver**
Override View.getLocationInWindow() to ensure it takes StaticTransformation (which Uno is using for RenderTransforms) into account.

This fixes:
 - placement of MenuFlyout anchored to view in transformed hierarchy
 - return of TransformToVisual for view in transformed hierarchy

GitHub Issue (If applicable): #

## PR Type

- Bugfix
What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

https://github.com/unoplatform/private/issues/37

### CODE REVIEW

**Comment by dr1rrb** [comment]
This is already fixed in https://github.com/unoplatform/uno/pull/1256

**Comment by dr1rrb** [comment]
Disregard my previous comment, I had to fix it for the pointer events, but not for the transform case. 

Did you check the [getLocationOnScreen](https://developer.android.com/reference/android/view/View.html#getLocationOnScreen(int[])) ?

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+			var windowHeight = int.Parse(windowHeightText);
+			var transformX = int.Parse(transform2XText);
+			var transformY = int.Parse(transform2YText);
+
+			Assert.Greater(windowWidth, 100);
+			Assert.Greater(windowHeight, 100);
+
+			Assert.AreEqual(windowWidth - borderWidth - translateX, transformX);
+			Assert.AreEqual(windowHeight - borderHeight - translateY, transformY);
+		}
```
Comment (jeromelaban) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs]: Should there be a screenshot at the end?

**Comment by davidjohnoliver** [comment]
> Disregard my previous comment, I had to fix it for the pointer events, but not for the transform case.
> 
> Did you check the [getLocationOnScreen](https://developer.android.com/reference/android/view/View.html#getLocationOnScreen(int%5B%5D)) ?

`getLocationOnScreen()` calls `getLocationOnWindow()`, so it will also get the fix.

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs (hunk 1 above)
Comment (davidjohnoliver) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs]: Sure, added one

**Pull request merged**